internal/ui/operations/details/details.go

```diff
@@ -56,6 +56,45 @@ func (s *Operation) Init() tea.Cmd {
 	return s.load(s.revision.GetChangeId())
 }
 
+func (s *Operation) preserveSelection(items []*item, currentHighlightedFile string) (context.SelectedItem, int) {
```

Comment:
im not sure if a whole function of `preserveSelection` is needed, since there's `context.AddCheckedItem` and `setItem`